providers/nedb-class.js

```diff
@@ -0,0 +1,170 @@
+const { resolve } = require("path");
+const { Collection } = require("discord.js");
+const { promisifyAll } = require("tsubaki");
+const Datastore = require("nedb-core");
+
+promisifyAll(Datastore.prototype);
+
+class NeDB {
+  /**
+   * Creates an instance of the NeDB class
+   * @param {string} [name="komada"] Name of the Datastore
+   * @param {boolean} [persistent=true] Is Persistent?
+   *
+   * @constructor NeDB
+   */
+  constructor(name = "komada", persistent = true) {
+    this.name = name;
+    this.persistent = persistent;
+    this.db = null;
+  }
+
+  /**
+   * Connects to the NeDB datastore and populates this.db
+   *
+   * @param {?Client} client Komada Client
+   * @returns {Datastore} The NeDB Datastore itself
+   *
+   * @memberof NeDB
```

Comment:
The memberof is not required. And you can remove the space that is between the description and param. Same for the rest of the file.